api/server/api.cc

```diff
@@ -135,7 +135,7 @@ bool API::action_nic_add(const app::Nic &nic, std::string *path,
     // Add NIC in model
     std::pair<std::string, app::Nic> p(nic.id, nic);
     app::model.nics.insert(p);
-    return true;
+    return path->size();
```

Comment:
the message is not very explicit, "return false on empty path"
In C I would use `return !! path->size()` but as bool is a type in C++, I guess the `!!` is not needed, but maybe more explicit ?